src/metpy/calc/thermo.py

```diff
@@ -3659,3 +3659,92 @@ def cross_totals(pressure, temperature, dewpoint):
 
     # Calculate vertical totals.
     return td850 - t500
+
+
+@exporter.export
+@preprocess_and_wrap()
+@check_units('[pressure]', '[temperature]', '[temperature]', '[speed]')
+def sweat_index(pressure, temperature, dewpoint, speed, direction):
+    """Calculate SWEAT Index.
+
+    SWEAT Index derived from [MillerBidnerMaddox1971]_ and
+    [MillerBidnerMaddox1972]_:
+    SWEAT = 12Td850 + 20(TT - 49) + 2f850 + f500 + 125(S + 0.2)
```

Comment:
Let's get Sphinx to properly format this as math:
```suggestion
    SWEAT Index derived from [Miller1972]:

    .. math:: SWEAT = 12Td850 + 20(TT - 49) + 2f850 + f500 + 125(S + 0.2)
```